index.bs

```diff
@@ -815,13 +859,16 @@ partial interface HTMLIFrameElement {
 
     <div class="algorithm" data-algorithm="construct-policy">
     Given an <a>ordered map</a> (|dictionary|) and an [=origin=] (|origin|), this
-    algorithm will return a <a for="permissions policy">declared policy</a>.
-    1. Let |policy| be an empty [=ordered map=].
-    1. [=map/For each=] |feature-name| → |value| of |dictionary|:
+    algorithm will return a [=/declared policy=].
+    1. Let |declarations| be an empty ordered map.
+    1. Let |reporting-config| be an empty ordered map.
+    1. [=map/For each=] |feature-name| → (|value|, |params|) of |dictionary|:
         1. If |feature-name| does not identify any recognized
           <a>policy-controlled feature</a>, then [=iteration/continue=].
         1. Let |feature| be the <a>policy-controlled feature</a> identified by
           |feature-name|.
+        1. If |params|["report-to"] exists, and is a string, then set
```

Comment:
Sometimes this spec uses `"report-to"` (code formatting) and other times (like this case) it doesn't. I'd aim to be consistent (probably using code formatting everywhere).